docs/authoring.md

```diff
@@ -65,23 +64,35 @@ Do this by adding the following to ``MyAuthoredComponent.csproj```
 </ItemGroup>
 ```
 
-**If you are going to write your own nuspec** then the CsWinRT target that adds the hosting dlls to your package will not run, and you should make sure your nuspec contains the following ```file``` entries for ```MyAuthoredComponent``` (note: your TargetFramework may vary).
+**If you are going to write your own nuspec, i.e. not rely on automatic packaging** then the CsWinRT target that adds the hosting dlls to your package will not run, and you should make sure your nuspec contains the following ```file``` entries for ```MyAuthoredComponent``` (note: your TargetFramework may vary). If adding a file entry for ```Coords.targets``` does not work, then update the project file per the above example. 
 
 ``` nuspec
-  <files>
-    <file src="build\MyAuthoredComponent.CsWinRT.targets"             target="build\MyAuthoredComponent.CsWinRT.targets" />
-    <file src="build\MyAuthoredComponent.targets"                     target="build\MyAuthoredComponent.targets" />
-    <file src="buildTransitive\MyAuthoredComponent.CsWinRT.targets"   target="buildTransitive\MyAuthoredComponent.CsWinRT.targets" />
-    <file src="buildTransitive\MyAuthoredComponent.targets"           target="buildTransitive\MyAuthoredComponent.targets" />
-    <file src="lib\native\MyAuthoredComponent.dll"                    target="lib\native\MyAuthoredComponent.dll" />
-    <file src="lib\native\Microsoft.Windows.SDK.NET.dll"              target="lib\native\Microsoft.Windows.SDK.NET.dll" />
-    <file src="lib\native\WinRT.Host.Shim.dll"                        target="lib\native\WinRT.Host.Shim.dll" />
-    <file src="lib\native\WinRT.Runtime.dll"                          target="lib\native\WinRT.Runtime.dll" />
-    <file src="lib\net5.0-windows10.0.19041\MyAuthoredComponent.dll"  target="lib\net5.0-windows10.0.19041\MyAuthoredComponent.dll" />
-    <file src="metadata\MyAuthoredComponent.winmd"                    target="metadata\MyAuthoredComponent.winmd" />
-    <file src="runtimes\win-x64\native\WinRT.Host.dll"                target="runtimes\win-x64\native\WinRT.Host.dll" />
-    <file src="runtimes\win-x86\native\WinRT.Host.dll"                target="runtimes\win-x86\native\WinRT.Host.dll" />
-  </files>
+<files>
+  <file src="$(TargetDir)Coords.dll"                    target="lib\native\Coords.dll" />
+  <file src="$(TargetDir)Microsoft.Windows.SDK.NET.dll" target="lib\native\Microsoft.Windows.SDK.NET.dll" />
+  <file src="Generated Files\Coords.winmd" target="winmd\Coords.winmd" />

```

Comment:
Is the winmd also copied to bin folder?  If so, might want to copy from there.

Reply:
No, it's not. Do we want to look into doing that?

Reply:
We should probably copy it to bin as part of the build as winmd can change later due to editing source and if they didn't do a build, the wrong version of the winmd would have got included.  But if you want, you can address that as a bug fix.

Reply:
It's okay, it was pretty quick.